elisp/intero.el

```diff
@@ -1280,7 +1297,8 @@ function is subsequently applied to each line, once."
             'intero-linkify-process-output
             t t)
   (setq-local comint-prompt-read-only t)
-  (add-hook 'completion-at-point-functions 'intero-completion-at-point nil t)
+  (setq-local company-idle-delay nil)
```

Comment:
This line is out of place: we shouldn't be overriding general `company` preferences arbitrarily. What's the intention?

Reply:
It fixed an annoying issue with my config. You're right that it's out of place, it should be in fixed on my end.